tests/system/ControllerTest.php

```diff
@@ -71,6 +71,7 @@ protected function setUp(): void
         $this->response    = new Response($this->config);
         $this->logger      = \Config\Services::logger();
         $this->codeigniter = new MockCodeIgniter($this->config);
+        $this->codeigniter->setContext('web');
```

Comment:
I think these are no longer needed with the default property?

Reply:
Removed.